cmd/kubeadm/app/cmd/phases/selfhosting.go

```diff
@@ -34,15 +34,33 @@ import (
 	configutil "k8s.io/kubernetes/cmd/kubeadm/app/util/config"
 	kubeconfigutil "k8s.io/kubernetes/cmd/kubeadm/app/util/kubeconfig"
 	"k8s.io/kubernetes/pkg/api/legacyscheme"
+	"k8s.io/kubernetes/pkg/util/normalizer"
+)
+
+var (
+	selfhostingLongDesc = normalizer.LongDesc(`
+		Converts static Pod files for control plane components into self-hosted DaemonSets configured via the Kubernetes API.
+
+		Self-hosting in 1.8 has some important limitations. See documentation.
```

Comment:
Self-hosting in v1.8 is on an alpha level with some limitations, but on a beta level in v1.9. See the documentation